geomesa-kafka/geomesa-kafka-confluent/src/test/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreFactoryTest.scala

```diff
@@ -0,0 +1,124 @@
+/***********************************************************************
+ * Copyright (c) 2013-2021 Commonwealth Computer Research, Inc.
```

Comment:
```suggestion
 * Copyright (c) 2013-2022 Commonwealth Computer Research, Inc.
```

sorry, you'll probably need to update this since i updated the year